### PR TITLE
Rails 7 update

### PR DESCRIPTION
**Pull request by sonalkr132**
blocked because delayed_job seems dead.

### CODE REVIEW

**Comment by simi** [comment]
@sonalkr132 would you welcome PR to move to another background job system like [que](https://github.com/que-rb/que)?

**Comment by sonalkr132** [comment]
someone has created fork for delayed_job with rails 7 support https://github.com/Betterment/delayed. At this point, I am hoping collectiveidea one comes back live.
Would sidekiq be overkill? It may be more sustainable.

**Comment by simi** [comment]
Que is DB based as well. Sidekiq needs Redis and has diferent outcomes regarding durability AFAIK.

**Comment by sonalkr132** [comment]
Yeah, adding redis would be an issue. resque uses same. Let's wait a week or two for collectiveidea. que has very few contributors. We can discuss and decide then.

**Comment by simi** [comment]
AFAIK que is very well long-term maintained project (https://www.ruby-toolbox.com/projects/que). It is the second most popular DB based Ruby queuing system especially crafted for PostgreSQL using a lot of PG specific stuff.

**Comment by morgoth** [comment]
Hey, instead of que, please do consider the https://github.com/bensheldon/good_job which is very well maintained and the author is always eager to help to solve any issues.

We're using this gem in production for some time and it's a much better experience for us than que was.

File: config/initializers/clearance.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require_relative "../../lib/confirmed_user_guard"
```
Comment (sonalkr132) [inline config/initializers/clearance.rb:1]: @simi do you have context on the need for this change? require_relative is not a huge issue but constant missing/module undefined keeps happening with every rails update.

Comment (sonalkr132) [reply config/initializers/clearance.rb:1]: The issue for us was in production env. docker build was failing on asset precompile:
```
#23 [build 12/13] RUN RAILS_ENV=production RAILS_GROUPS=assets SECRET_KEY_BASE=1234 bin/rails assets:precompile
#23 4.651 rails aborted!
#23 4.651 NameError: uninitialized constant Lograge::Formatters::Datadog
#23 4.651 /app/config/initializers/lograge.rb:7:in `block in <main>'
```
https://github.com/rubygems/rubygems.org/runs/4872242431?check_suite_focus=true#step:6:512

I was wondering if we could identify the real issue, we could use the recommended fix. I guess require_relative will do for now. We have use same fix all the previous times it broke https://github.com/rubygems/rubygems.org/pull/2598

**Pull request merged**